tests/contract/test_type_configuration.py

```diff
@@ -20,14 +20,15 @@ def setup_function():
     # Resetting before each test
     TypeConfiguration.TYPE_CONFIGURATION = None
 
+def teardown_function():
+    # Rest after to clean TypeConfiguration before the next test
+    TypeConfiguration.TYPE_CONFIGURATION = None
+
 
 def test_get_type_configuration_with_not_exist_file():
     with patch("builtins.open", mock_open()) as f:
         f.side_effect = FileNotFoundError()
-        try:
-            TypeConfiguration.get_type_configuration()
-        except FileNotFoundError:
-            pass
+        assert TypeConfiguration.get_type_configuration() is None
```

Comment:
These tests don't even make sense as written.  `FileNotFoundError` is suppressed